frequency_calculator.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+from typing import Annotated
+
+import typer
+from typing_extensions import Optional
+
+from characters_counter import count_character_frequency
+from logging_config import setup_logging
+
+app = typer.Typer()
+logger = setup_logging()
+
+
+@app.command()
+def calculate(
+    file: Annotated[Optional[Path], typer.Option("--file", "-f")] = None,
```

Comment:
Optional is deprecated feature, please review one of my previous comments in the previous PRs

Reply:
Oh, yeah you pointed out me this in previous reviews... In my defense, I read a documentation and this annotation was used throughtout. And without second thought I repeated it...